saleor/graphql/menu/mutations.py

```diff
@@ -254,9 +280,12 @@ def clean_operation(operation: _MenuMoveOperation):
             if operation.menu_item.is_ancestor_of(operation.parent):
                 raise ValidationError(
                     {
-                        "parent": (
-                            "Cannot assign a node as child of "
-                            "one of its descendants."
+                        "parent": ValidationError(
+                            (
+                                "Cannot assign a node as child of "
+                                "one of its descendants."
+                            ),
+                            code=MenuErrorCode.CANNOT_ASSIGN_NOTE,
```

Comment:
Looks like a typo, should be `CANNOT_ASSIGN_NODE`? But I think this one and `ASSIGN_MENU_ITEM_TO_ITSELF` are in fact the same error which tells you that this particular assignment is not allowed.

Reply:
Yes, it was a typo.
I've unified those two error codes.